#goal definition
std_msgs/String folder_path
#int32 images_number

---
#result definition
bool finished
int32 n_imgs1
int32 n_imgs2
---
#feedback
int32 n_imgs1
int32 n_imgs2
